src/java.base/share/classes/java/lang/module/ModuleDescriptor.java

```diff
@@ -2627,7 +2627,7 @@ private static <M> String toString(Set<M> mods, String what) {
     private static int modsHashCode(Iterable<? extends Enum<?>> enums) {
         int h = 0;
         for (Enum<?> e : enums) {
-            h = h * 43 + Objects.hashCode(e.name());
+            h = h + Objects.hashCode(e.name()) * 43;
```

Comment:
why * 43 ? Isn't String.hashCode good enough?

Reply:
A sum of e.name().hashCode() should be okay here.